src/libp2p.js

```diff
@@ -397,8 +392,6 @@ const createLibp2p = async ({
   hostAddrs,
   announceAddrs,
   dht,
-  dhtClient,
-  connMgr,
```

Comment:
Were these lines removed for a reason? Other than that, this looks good!

Reply:
@jacobheun I removed them since they weren't used. Thought it was good code hygiene. If you want, I can put them back :)

Reply:
Nope that's fine, just wanted to make sure :)